dotnet-desktop-guide/net/wpf/whats-new/net90.md

```diff
@@ -0,0 +1,175 @@
+5. When the `ThemeMode` is set to a Window, it will take precedence over the Application's `ThemeMode`. In case Window `ThemeMode` is set to None, the window will adhere to Application's `ThemeMode`, even if Application uses Fluent Theme.
+6. The default value of `ThemeMod`e is None.
+
+In addition to these behaviors, the ThemeMode is also designed to respect the Fluent Theme Dictionary added to the Application or Window. AFluent Themes can also be loaded by including the respective Fluent Dictionary. If the given application or window is loaded with a given Fluent Dictionary, let's say Light, then the ThemeMode will be synced to Light Mode as well and vice-versa.
+
+This can be enabled by adding the following to your App.xaml:
+
+```xml
+<Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <ResourceDictionary Source="pack://application:,,,/PresentationFramework.Fluent;component/Themes/Fluent.xaml" />
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+</Application.Resources>
+```
+
+## AccentColors as SystemColors
+
+Since Windows 10, Accent Color became a cornerstone for visual styling, enhancing consistency across apps by aligning with the system theme. The latest introduction of AccentColors and corresponding brushes in SystemColors simplifies this process for WPF developers, offering them direct access to system accent colors and their variations.
+
+This streamlines the workflow, ensuring apps can seamlessly integrate system-defined aesthetics, ultimately improving the user experience without extra coding overhead. The new API offers a more reliable and efficient way to deliver visually cohesive applications.
+
+1. **Colors**: The following System.Windows.Media.Color are being introduced corresponding to the current accent color of the system and its primary, secondary and tertiary variations in both Light and Dark mode.
+
+    ```cs
+    AccentColor
+    AccentColorLight1
+    AccentColorLight2
+    AccentColorLight3
+    AccentColorDark1
+    AccentColorDark2
+    AccentColorDark3
+    ```
+
+2. **ResourceKey**: Similary, the following System.Windows.ResourceKey are being introduced.
+
+    ```cs
+    AccentColorKey
+    AccentColorLight1Key
+    AccentColorLight2Key
+    AccentColorLight3Key
+    AccentColorDark1Key
+    AccentColorDark2Key
+    AccentColorDark3Key
+    ```
+
+3. **SolidColorBrush**: Similary, the following System.Windows.Media.SolidColorBrush are being introduced.
+
+    ```cs
+    AccentColorBrush
+    AccentColorLight1Brush
+    AccentColorLight2Brush
+    AccentColorLight3Brush
+    AccentColorDark1Brush
+    AccentColorDark2Brush
+    AccentColorDark3Brush
+    ```
+
+### Usage of the `AccentColor` APIs
+
+Usage of the AccentColor APIs, update the applicable properties as below:
+
+```xml
+<Button Content="Sample WPF Button" Background="{x:Static SystemColors.AccentColorBrush}" />

```

Comment:
This usage is not right, it throw's Argument Null exception when the AccentColor is changed in the system. This should be replaced with the following :

```xml
<Button Content="Sample WPF Button" Background="{DynamicResource {x:Static SystemColors.AccentColorBrushKey}}" />
```